conan/tools/env/environment.py

```diff
@@ -0,0 +1,291 @@
+import fnmatch
+import os
+import textwrap
+from collections import OrderedDict
+
+from conans.errors import ConanException
+from conans.util.files import save
+
+
+class _EnvVarPlaceHolder:
+    pass
+
+
+class _Sep(str):
+    pass
+
+
+class _PathSep:
+    pass
+
+
+def environment_wrap_command(filename, cmd, cwd=None):
+    assert filename
+    filenames = [filename] if not isinstance(filename, list) else filename
```

Comment:
```suggestion
        try:
            from collections.abc import Iterable
        except ImportError:
            from collections import Iterable
    filenames = [filename] if not isinstance(filename, Iterable) else filename
```

Reply:
We have `conans.utils.misc.make_tuple` that will work here (pay attention, a string is iterable)